README.md

```diff
@@ -12,11 +16,15 @@ git clone https://github.com/surecart/wordpress-sdk.git licensing
 Now include the dependencies in your plugin/theme.
 
 ```php
-if ( ! class_exists( 'SureCart\Licensing\Client' ) ) {
-    require_once __DIR__ . '/licensing/src/Client.php';
-}
+require_once __DIR__ . '/licensing/SureCartSdkLoader.php';
```

Comment:
Did you test this with this sdk in 2 plugin directories? I would assume this may fail because they are being required in 2 different places maybe. But I'm not sure.